src/ios/CDVUIInAppBrowser.m

```diff
@@ -779,10 +781,17 @@ - (void)createViews
         [self.toolbar setItems:@[self.closeButton, flexibleSpaceButton, self.backButton, fixedSpaceButton, self.forwardButton]];
     }
 
-    self.view.backgroundColor = [UIColor grayColor];
+    self.view.backgroundColor = IsAtLeastiOSVersion(@"13.0") ? [UIColor systemBackgroundColor] : [UIColor whiteColor];
     [self.view addSubview:self.toolbar];
     [self.view addSubview:self.addressLabel];
     [self.view addSubview:self.spinner];
+
+    UIEdgeInsets insets = [self getWindowSafeAreaInsets];
+    self.backgroundView = [[UIView alloc] initWithFrame:CGRectMake(0, insets.top, self.view.bounds.size.width, self.view.bounds.size.height - insets.top - insets.bottom)];
+    self.backgroundView.autoresizingMask = (UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight);
+    self.backgroundView.backgroundColor = [UIColor grayColor];
```

Comment:
What does this color affect and why is it hardcoded to gray?

Reply:
Previously that was the background color of the whole screen. You can see the gray in the bar that contains the URL. I just shifted it around.